code/game/gamemodes/blob/blob_finish.dm

```diff
@@ -11,18 +11,21 @@
 /datum/game_mode/blob/declare_completion()
 	if(blobwincount <= blobs.len)
 		feedback_set_details("round_end_result","blob win - blob took over")
+		GLOB.gamemode_end_state = "BLOB-BLOB-WIN"

```

Comment:
Would a define be better here and in other places than a string?

Reply:
True, but if you wanted to change the define later (or if someone accidentally changed one of these) it would be much easier to change and fix